app/api/zora/tokens/route.ts

```diff
@@ -0,0 +1,23 @@
+import { EVENT_SETUP_NEW_TOKEN, EVENT_ZORA_TOKENS } from '@/lib/consts';
+import { createStackClient } from '@/lib/stack/client';
+import trackEndpoint from '@/lib/stack/trackEndpoint';
+import { NextRequest } from 'next/server';
+
+const stack = createStackClient(3340)
```

Comment:
can you set the `magic number` to the `const` lib?
- example: `export const TOKEN_INDEXER_POINT_ID=3340`